keyboards/handwired/nanggong/info.json

```diff
@@ -0,0 +1,111 @@
+{
+    "manufacturer": "Sanma",
+    "keyboard_name": "Nanggong",
+    "maintainer": "Sanma",
+    "bootloader": "caterina",
```

Comment:
```suggestion
    "development_board": "promicro",
```

Reply:
This still needs changing

Reply:
thanks sir for look and review my PR again, I've adjusted the changes as the review instructions, so whenever you get a chance, could you please take another peek? thanks a lot